rust-toolchain.toml

```diff
@@ -1,5 +0,0 @@
-[toolchain]
-channel = "nightly-2021-11-29"
-components = [ "rustfmt", "rustc", "rust-std", "cargo", "clippy", "llvm-tools-preview"]
-targets = [ "wasm32-unknown-unknown" ]
-profile = "minimal"
```

Comment:
Please don't remove this. This will break the local env for developers. At most use a different one for a CI run.